homework/Vasili_Zasinets/Shape/src/main/java/com/tutrit/java/quickstart/shape/Square.java

```diff
@@ -0,0 +1,39 @@
+package com.tutrit.java.quickstart.shape;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Square extends AbstractShape {
+
+    private final double sideLength;
+
+    Logger log = LoggerFactory.getLogger("Square");
+
+    public Square(String name, double sideLength) {
+        super(name);
+        this.sideLength = sideLength;
+    }
+
+    @Override
+    public double getArea() throws ShapeLengthException {
+        if (sideLength <= 0) {throw new ShapeLengthException("Incorrect data entered");}
+        else {return sideLength * sideLength;}
```

Comment:
closing curly brace should be on the same row with else 
return should be on the next row after curly brace.

Reply:
fixed